ansys/mapdl/core/commands.py

```diff
@@ -20,6 +20,159 @@
     inq_func
 )
 
+import re
+import numpy as np
+
+try:
+    import pandas as pd
```

Comment:
This needs to be a lazy load. If we load pandas when initially loading `ansys-mapdl-core`, it's going to marginally slower, even if there's no intention to load `pandas`.